### PR TITLE
Make sure CI runs on every PR, even if stacked

### PR DESCRIPTION
**Pull request by andresilva91**
## Motivation

Stacked PRs were not triggering CI

## Proposal

Make sure all PRs, including stacked ones, trigger CI

## Test Plan

CI

- [reviewer checklist](https://github.com/linera-io/linera-protocol/blob/main/CONTRIBUTING.md#reviewer-checklist)

### CODE REVIEW

**Comment by andresilva91** [comment]
Current dependencies on/for this PR:
* main
  * **PR #1062** <a href="https://app.graphite.dev/github/pr/linera-io/linera-protocol/1062" target="_blank">Graphite</a> 
    * **PR #1073** <a href="https://app.graphite.dev/github/pr/linera-io/linera-protocol/1073" target="_blank">Graphite</a> 
      * **PR #1078** <a href="https://app.graphite.dev/github/pr/linera-io/linera-protocol/1078" target="_blank">Graphite</a> 
        * **PR #1079** <a href="https://app.graphite.dev/github/pr/linera-io/linera-protocol/1079" target="_blank">Graphite</a>  👈
          * **PR #1075** <a href="https://app.graphite.dev/github/pr/linera-io/linera-protocol/1075" target="_blank">Graphite</a> 

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/linera-io/linera-protocol/1079?utm_source=stack-comment).

**Comment by ma2bd** [comment]
Importantly, `publish-docs` was skipped so this should be good.

**Review by andresilva91 — COMMENTED** [review]
This is the base commit, so we should be good to land this before I fix the issues with the 2 PRs stacked on top of this

File: .github/workflows/dynamodb.yml [hunk 1]
```diff
@@ -2,9 +2,11 @@ name: DynamoDB tests
 
 on:
   push:
-    branches: [ main ]
+    branches:
+      - "**"
   pull_request:
-    branches: [ main ]
+    branches:
+      - "**"
```
Comment (andresilva91) [inline .github/workflows/dynamodb.yml:9]: Should we have just one of these here then, and not both?

**Review by ma2bd — APPROVED** [review]
LGTM

**Pull request merged**